blocks/bricks/simple.py

```diff
@@ -284,6 +284,28 @@ def apply(self, input_):
         return tensor.switch(input_ > 0, input_, 0)
 
 
+class LeakyRectifier(Activation):
+    r"""A linear transformation with optional bias.
```

Comment:
Seems like copy-paste from somewhere else. It's totally not true.

I think, documentation needs some citations.